src/mon/Monitor.cc

```diff
@@ -2831,7 +2831,17 @@ void Monitor::do_health_to_clog(bool force)
       summary == health_status_cache.summary &&
       level == health_status_cache.overall)
     return;
-  clog->health(level) << "overall " << summary;
+
+  if (summary != health_status_cache.summary &&
+      level != HEALTH_OK) {
+    string plain;
+    level = healthmon()->get_health_status(true, nullptr, &plain);
+
```

Comment:
could you drop this empty line? see https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace 

i don't think this empty line separates two parts of code serving different purposes.